server/src/metadata/tenant-initialisation/standard-objects/activity-target/activity-target.metadata.json

```diff
@@ -0,0 +1,44 @@
+{
+  "nameSingular": "activityTargetV2",
+  "namePlural": "activityTargetsV2",
+  "labelSingular": "Activity Target",
+  "labelPlural": "Activity Targets",
+  "targetTableName": "activity_target",
+  "description": "An activity target",
+  "icon": null,
+  "fields": [
+    {
+      "type": "text",
+      "name": "id",
+      "label": "ID",
+      "targetColumnMap": {
+        "value": "id"
```

Comment:
Small note here: We have implemented "basic fields" by default in our code which means primary keys and dates (id, createdAt, updatedAt, deletedAt) are not needed here

Reply:
Which also means we don't need to store them as FieldMetadata nor TenantMigration. The TenantMigration that creates a table will automatically create them and the dynamic API will automatically expect that they exist so it won't try to fetch them from FieldMetadata